src/components/CartProductCard/CartProductCard.tsx

```diff
@@ -0,0 +1,52 @@
+import { Phone } from '@/types/Phone';
+import { FC, useState } from 'react';
+import styles from './styles.module.scss';
+import classNames from 'classnames';
+interface INCartProductCard {
+  phone: Phone;
+}
+
+export const CartProductCard: FC<INCartProductCard> = ({ phone }) => {
+  const [count, setCount] = useState(1);
+
+  return (
+    <article className={styles.card}>
+      <div className={styles.containerDeleteImgTitle}>
+        <div className={styles.containerCancelImg}>
+          <button className={styles.deleteButton}></button>
+          <a href="#">
+            <img
+              src={phone.images[0]}
+              alt={phone.name}
+              className={styles.picture}
+            />
+          </a>
+        </div>
+        <a href="#">
+          <h2 className={styles.title}>{phone.name}</h2>
+        </a>
+      </div>
+
+      <div className={styles.containerButtonsPrice}>
+        <div className={styles.buttons}>
+          <button
```

Comment:
In my PR with footer I created a button component with an icon, in fact there it is enough to add styles for each of the icons and reuse as many times as you want for this kind of round buttons

[You can see it here](https://github.com/FS-AUG24-TEAM-4/group_project/blob/develop/src/components/IconButton/IconButton.tsx)